docs/AsyncWrap/README.md

```diff
@@ -131,25 +152,33 @@ asyncWrap.disable();
 
 #### The Hooks
 
-Currently there are 3 hooks: `init`, `before` and `after`. The function
-signatures are quite similar. The `this` variable refers to the handle object,
-and `init` hook has two extra arguments `provider` and `parent`.
+Currently there are 4 hooks: `init`, `pre`, `post` `destroy`. The `this`
+variable refers to the handle object. The `init` hook has three extra arguments
```

Comment:
May want to make a note that the associated JS object can usually be accessed via `this.owner`. This isn't official, but is the common practice. Maybe we want to solidify this? /cc @bnoordhuis